client/stylesheets/_toCat.scss

```diff
@@ -4813,6 +4810,71 @@ On click the images will be displayed at normal size to complete the effect
 }
 
 
+/*Options DropDown*/
+
+.optionsClick {
+	margin:100px auto;
+	width:200px;
+}
+
+.optionsClickv ul {
+	position:relative;
+	font-weight:900;
+}
+
+.optionsClick ul li {
+	position:relative;
+	list-style:none;
+	cursor:pointer;
+}
+
+.optionsClick ul li ul {
+	position:absolute;
+	left:0;
+	right:0;
+}
+
+.optionsClick ul .clicker {
+	position:relative;
+	background:#2284B5;
+	color:#FFF;
+}
+
+.optionsClick ul .clicker:hover,.click-nav ul .active {
+	background:#196F9A;
```

Comment:
Color literals like `#196F9A` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#196F9A` should be written as `#196f9a`<br>Line should be indented with spaces, not tabs<br>Colon after property should be followed by one space